internal/cli/atlas_clusters_list.go

```diff
@@ -27,22 +27,25 @@ type atlasClustersListOpts struct {
 	*globalOpts
 	pageNum      int
 	itemsPerPage int
-	store        store.ClusterLister
+	storeAT      store.ClusterLister
+	storeOM      store.ListAllClusters
 }
 
 func (opts *atlasClustersListOpts) init() error {
+	var err error
+
 	if opts.ProjectID() == "" {
-		return errMissingProjectID
+		opts.storeOM, err = store.New()
+	} else {
+		opts.storeAT, err = store.New()
 	}
 
-	var err error
-	opts.store, err = store.New()
 	return err
 }
 
-func (opts *atlasClustersListOpts) Run() error {
+func (opts *atlasClustersListOpts) RunAT() error {
```

Comment:
this all comes from the same thing, multiple properties instead of a more flexible contract, there's no need to have multiple `Run` methods